spec/requests/settings_spec.rb

```diff
@@ -7,16 +7,10 @@
            })
   end
   let(:retreived_tenant) { Tenant.find(tenant.id) }
-  let(:rbac) { instance_double(RBAC::Access) }
-
-  before do
-    allow(RBAC::Access).to receive(:new).and_return(rbac)
-    allow(rbac).to receive(:process).and_return(rbac)
-  end
 
   context "when the user is a catalog admin" do
     before do
-      allow(rbac).to receive(:catalog_admin?).and_return(true)
+      allow(RBAC::Roles).to receive(:assigned_role?).and_return(true)
```

Comment:
should probably add `.with("Catalog Administrator")` to this and the below spec setups.